components/service/glean/src/main/java/mozilla/components/service/glean/StringListMetricType.kt

```diff
@@ -34,14 +34,10 @@ data class StringListMetricType(
     // Holds the Job returned from launch{} for awaiting purposes
     private var ioTask: Job? = null
 
-    companion object {
-        // Maximum length of any passed value string, in characters.
-        const val MAX_STRING_LENGTH = 50
-    }
-
     /**
      * Appends a string value to one or more string list metric stores.  If the string exceeds the
-     * maximum string length, defined as [MAX_STRING_LENGTH], it will be truncated.
+     * maximum string length, defined as [StringListsStorageEngine.MAX_STRING_LENGTH], it will be
+     * truncated by the storage engine.
```

Comment:
We don't need to mention the storage engine, we can just terminate with "it will be truncated". The storage engine is an implementation detail that doesn't need to show up in the docs for the developers within the IDE.